src/main/java/thePackmaster/powers/siegepack/ShellPower.java

```diff
@@ -0,0 +1,77 @@
+package thePackmaster.powers.siegepack;
+
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.powers.AbstractPackmasterPower;
+
+import java.util.Objects;
+
+//This buff is the actual shells, boosting damage and consumed (1 per attack).
+public class ShellPower extends AbstractPackmasterPower {
+    public static final String POWER_ID = SpireAnniversary5Mod.makeID(ShellPower.class.getSimpleName());
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public int DAMAGE_BOOST = 5;
+
+    public ShellPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, PowerType.BUFF, false, owner, amount);
+        amount2 = updateBoostValue();
+        updateDescription();
+    }
+
+    public float atDamageGive(float damage, DamageInfo.DamageType type) {
+        return type == DamageInfo.DamageType.NORMAL ? damage + (float)updateBoostValue() : damage;
+    }
+
+    public void onUseCard(AbstractCard card, UseCardAction action) {
+        if (card.type == AbstractCard.CardType.ATTACK) {
+            this.flash();
+            //Remove only 1 stack, don't change damage.
+            this.addToBot(new ReducePowerAction(this.owner,this.owner, this, 1));
+        }
+    }
+
+    public void onApplyPower(AbstractPower power, AbstractCreature target, AbstractCreature source) {
```

Comment:
where is this called from?

Reply:
Well this is odd, I realize I forgot the @Override, but I didn't notice any malfunction without it. I've added it now and will run some more tests